src/makeRobotAccountant.js

```diff
@@ -7,6 +7,23 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  const getSum1 = function(num1) {
+    count += 1;
```

Comment:
```suggestion
    count++;
```

Reply:
I prefer it doing +=, anyways the meaning is same.